exchange/exchange.go

```diff
@@ -1622,3 +1624,17 @@ func setSeatNonBid(bidResponseExt *openrtb_ext.ExtBidResponse, seatNonBidBuilder
 	bidResponseExt.Prebid.SeatNonBid = seatNonBidBuilder.Slice()
 	return bidResponseExt
 }
+
+func isEEACountry(country string, eeaCountries []string) bool {
+	if len(eeaCountries) == 0 {
+		return false
+	}
+
+	country = strings.ToUpper(country)
+	for _, c := range eeaCountries {
+		if strings.ToUpper(c) == country {
+			return true
+		}
+	}
```

Comment:
Changes look good. 
What do you think about storing `eeaCountries` in `map[string]struct{}`, so the country search will be `eeaCountries[strings.ToUpper(country)]` ?
All countries should be stored in uppercase.